extensions/azurecore/src/azurecore.d.ts

```diff
@@ -76,10 +83,18 @@ declare module 'azurecore' {
 		provideResources(): azureResource.IAzureResourceProvider[];
 
 		runGraphQuery<T extends azureResource.AzureGraphResource>(account: azdata.Account, subscriptions: azureResource.AzureResourceSubscription[], ignoreErrors: boolean, query: string): Promise<ResourceQueryResult<T>>;
+		makeHttpGetRequest(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, ignoreErrors: boolean, url: string): Promise<HttpGetRequestResult>;
```

Comment:
Let's not add this to the public API for now unless we have a good reason for using it.

Exposing the graph API is useful since we can combine queries - but you can't really do that with the REST calls so I'd prefer we just keep those ones as getXXXX functions here.